### PR TITLE
(Chore) Update packages

### PR DESCRIPTION
**Pull request by aditudorache**
This PR updates the packages with minor and patch version. The packages with major version are not entirely updated, The webpack and the dependent packages are not yet updated, for this change there should be more time reserved.

### CODE REVIEW

**Review by MauritsR — CHANGES_REQUESTED** [review]
Package lock changes look good, although I didn't expect the linting changes - left some comments on those

File: .eslintrc.js [hunk 1]
```diff
@@ -49,6 +49,8 @@ module.exports = {
             ignoreStatic: true,
           },
         ],
+        '@typescript-eslint/object-curly-spacing': 'off',
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply .eslintrc.js:52]: Enabled after discussion.

File: .eslintrc.js [hunk 2]
```diff
@@ -146,5 +148,7 @@ module.exports = {
     'jest/require-to-throw-message': 'off',
     'jest/valid-expect': 'off',
     'jest/valid-title': 'off',
+    'react/jsx-newline': 'off',
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply .eslintrc.js:151]: Disabled after discussion.

File: .eslintrc.js [hunk 3]
```diff
@@ -146,5 +148,7 @@ module.exports = {
...
+    'react/jsx-no-constructed-context-values': 'off',
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply .eslintrc.js:152]: Disabled for jest files

**Review by MauritsR — APPROVED** [review]

**Pull request merged**